src/app/components/coach-marks/coach-marks-simple-obs/coach-marks-simple-obs.component.scss

```diff
@@ -0,0 +1,63 @@
+
+.coachmark-backdrop {
+    .toggle-wrapper {
+        display: flex;
+        align-items: center;
+        position: absolute;
+        right: calc(15px + env(safe-area-inset-right));
+        top: calc(45px + env(safe-area-inset-bottom));
+ion-toggle {
+        opacity: 1;
+}
+
+ion-toggle::part(track) {
+    box-shadow: 0 0 5px 2px #fff;
+    }
+    }
```

Comment:
Her er det noe rart med innrykk. Eller er det en } for mye?

Reply:
done